Angular InstantSearch/getting-started/src/app/app.component.ts

```diff
@@ -1,4 +1,10 @@
 import { Component } from '@angular/core';
+import algoliasearch from 'algoliasearch/lite';
```

Comment:
inconsistent prettier formatting here, the other ones don't set single quote

Reply:
It is consistent with the file. We need to run Prettier on another PR.